test/get-predeployed-accounts.test.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from "chai";
+import { starknet } from "hardhat"
+
+describe("Devnet restart", function() {
+    this.timeout(900_000);
+
+    it("should pass", async () => {
+        const response = await starknet.devnet.getPredeployedAccounts();
+        expect(response).to.be.an('array')
+        expect(response[0].address).to.be.a('string')
+        expect(response[0].initial_balance).to.be.a('number')
+        expect(response[0].private_key).to.be.a('string')
+        expect(response[0].public_key).to.be.a('string')
+    });
+})
```

Comment:
GitHub is reporting a missing newline here.

Reply:
I add a new line at the end [48fde15](https://github.com/Shard-Labs/starknet-hardhat-example/pull/66/commits/48fde15de89778456fb896baec7b05938e12a5b2)